app/source.js

```diff
@@ -666,9 +663,10 @@ window.Source = function(templates, cwd, tm, source, revlayers, examples, isMapb
             success: function(metadata) {
                 // Clear loading state
                 $('#full').removeClass('loading');
-                var center = metadata.center;
-                var zoom = Math.max(metadata.minzoom, view.model.get('minzoom'));
-                map.setView([center[1], center[0]], zoom);
+                map.fitBounds([
```

Comment:
to match account for https://github.com/mapbox/mapbox-studio/pull/1270 - fitBounds should disable animation as well.